R/module_bookmark_manager.R

```diff
@@ -53,149 +29,99 @@
 #' @name module_bookmark_manager
 #' @aliases bookmark bookmark_manager bookmark_manager_module
 #'
-
 #' @rdname module_bookmark_manager
 #' @keywords internal
 #'
-bookmark_manager_ui <- function(id) {
+bookmark_module_ui <- function(id, modules) {
   ns <- NS(id)
-  div(
-    class = "manager_content",
-    div(
-      class = "manager_table_row",
-      span(tags$b("Bookmark manager")),
-      actionLink(ns("bookmark_add"), NULL, icon = suppressMessages(icon("solid fa-bookmark")), title = "add bookmark"),
-      NULL
-    ),
-    uiOutput(ns("bookmark_list"))
+
+  is_unbookmarkable <- rapply(
+    get_teal_bookmarkable_summary(modules),
+    Negate(isTRUE),
+    how = "unlist"
+  )
```

Comment:
If the button is created in the server, this part can be removed and the ui function need not take the `modules` argument.

Reply:
done